package.json

```diff
@@ -61,7 +49,10 @@
     "release": "npm run gulp -- release"
   },
   "main": "./src/index.js",
-  "browser": "./src/index.js",
```

Comment:
Thanks, that would have been an awesome PR ❤️

Reply:
yeah I can break this out, I noticed that crypto was showing up in the build so this prevents it